typescript/sdk/src/fee/EvmTokenFeeReader.hardhat-test.ts

```
@@ -0,0 +1,167 @@
+import { MultiProvider } from '../providers/MultiProvider.js';
+import { randomInt } from '../test/testUtils.js';
+import { normalizeConfig } from '../utils/ism.js';
+
+import { EvmTokenFeeDeployer } from './EvmTokenFeeDeployer.js';
+import { EvmTokenFeeReader } from './EvmTokenFeeReader.js';
+import { TokenFeeConfig, TokenFeeConfigSchema, TokenFeeType } from './types.js';
+
+export const MAX_FEE =
+  1157920892373161954235709850086879078532699846656405640394n;
+export const HALF_AMOUNT =
+  5789604461865809771178549250434395392663499233282028201970n;
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)